assets/components/switchable/switchable.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ----- Types ----- //
+
+export type Switch = 'On' | 'Off' | 'Error';
```

Comment:
I'd be inclined to keep this binary as switches will likely just be on or off. In the case where we decide we need to take some sub-action based on one switch, that will probably be dictated by a second switch, but those scenarios may be rare. In the case we're looking at here, the distinction is: The service is available / The service is unavailable. If it's unavailable we should see a message to that effect.